File: trunk/MetroApp/HrmMath/Evaluation/Spectrum/LFPercents.cs
using HrmMath.Data;

namespace HrmMath.Evaluation.Spectrum
{
    public class LFPercents : IEvaluation<double>
    {
        public double Evaluate(SessionData training)
        {
            var lf = training.Evaluate( new LF() );
            var tp = training.Evaluate( new TP() );
            return (lf / tp) * 100;
        }
    }
}

File: PolarMath/src/org/macondo/polar/evaluation/spectrum/LFPercents.java
package org.macondo.polar.evaluation.spectrum;

import org.macondo.polar.data.Training;
import org.macondo.polar.evaluation.Evaluation;

public class LFPercents implements Evaluation<Double> {
	
	public Double evaluate(Training training) {
		double lf = training.evaluate(new LF());
		double tp = training.evaluate(new TP());
		return (lf / tp) * 100;
	}

}
